fusor-ember-cli/app/controllers/configure-environment.js

```diff
@@ -33,8 +33,7 @@ export default Ember.Controller.extend(ConfigureEnvironmentMixin, NeedsDeploymen
 
   envNameValidator: AllValidator.create({
     validators: [
-      PresenceValidator.create({}),
-      AlphaNumericDashUnderscoreValidator.create({})
+      PresenceValidator.create({})
```

Comment:
Since you're just doing a single validator now, you can get rid of the AllValidator now and just do 

```javascript
envNameValidator: PresenceValidator.create({});
```